src/xrpld/app/misc/WasmHostFuncWrapper.cpp

```diff
@@ -1102,133 +1237,4 @@ class MockInstanceWrapper
     }
 };
 
-namespace test {
-bool
-testGetDataIncrement()
```

Comment:
Where is it moved?

Reply:
As our previous experience this error will be hard to find. Better leave it

Reply:
Ok, brought it back but excluded it from codecov